backend/api/serializers.py

```diff
@@ -85,15 +85,15 @@ class ExecutionRequestSerializer(Serializer):
 
     Attributes:
         timeout (int): Timeout for the API deployment, maximum value can be 300s.
-            If -1 it corresponds to async execution. Defaults to -1
+            If 0 it corresponds to async execution. Defaults to 0
```

Comment:
@chandrasekharan-zipstack should the default be 0 or the maximum value?

Reply:
@ritwik-g in the previous implementation it defaulted to async execution - I let it remain the same. If we're changing this behaviour we'd have to inform users this as a breaking change

Reply:
@chandrasekharan-zipstack if that is the case then isn't removing `-1` also a breaking change? Should we then keep the min as `-1` and default as `-1`

Reply:
@ritwik-g I wonder why I didn't catch that - let me revert the change if we're worried about this breaking change. We can make it later if necessary